packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -125,6 +96,45 @@ This is Twig implementation of the [Accordion] component.
 </Accordion>
 ```
 
+## Slot
+
+You can pass custom content (including HTML) into the slot inside Accordion
+Header:
+
+```html
+<AccordionHeader id="AccordionItemExample1Header" for="AccordionItemExample1Content" slot="<span>my content</span>">
```

Comment:
```suggestion
<AccordionHeader id="AccordionItemExample1Header" for="AccordionItemExample1Content" UNSAFE_slot="<span>my content</span>">
```

Reply:
There is an agreement that every prop that uses a `raw` filter will be prefixed with `UNSAFE_`.